Clip remove rebase

futchitwo opened the pull request:
<!-- ℹ お読みください / README
PRありがとうございます！ PRを作成する前に、コントリビューションガイドをご確認ください:
Thank you for your PR! Before creating a PR, please check the contribution guide:
https://github.com/misskey-dev/misskey/blob/develop/CONTRIBUTING.md
-->

# What
<!-- このPRで何をしたのか？ どう変わるのか？ -->
<!-- What did you do with this PR? How will it change things? -->

# Why
<!-- なぜそうするのか？ どういう意図なのか？ 何が困っているのか？ -->
<!-- Why do you do it? What are your intentions? What is the problem? -->

# Additional info (optional)
<!-- テスト観点など -->
<!-- Test perspective, etc -->

tamaina committed:
refactor: use Vite to build instead of webpack (#8575)

* update stream.ts

* https://github.com/misskey-dev/misskey/pull/7769#issuecomment-917542339

* fix lint

* clean up?

* add app

* fix

* nanka iroiro

* wip

* wip

* fix lint

* fix loginId

* fix

* refactor

* refactor

* remove follow action

* clean up

* Revert "remove follow action"

This reverts commit defbb416480905af2150d1c92f10d8e1d1288c0a.

* Revert "clean up"

This reverts commit f94919cb9cff41e274044fc69c56ad36a33974f2.

* remove fetch specification

* renoteの条件追加

* apiFetch => cli

* bypass fetch?

* fix

* refactor: use path alias

* temp: add submodule

* remove submodule

* enhane: unison-reloadに指定したパスに移動できるように

* null

* null

* feat: ログインするアカウントのIDをクエリ文字列で指定する機能

* null

* await?

* rename

* rename

* Update read.ts

* merge

* get-note-summary

* fix

* swパッケージに

* add missing packages

* fix getNoteSummary

* add webpack-cli

* :v:

* remove plugins

* sw-inject分離したがテストしてない

* fix notification.vue

* remove a blank line

* disconnect intersection observer

* disconnect2

* fix notification.vue

* remove a blank line

* disconnect intersection observer

* disconnect2

* fix

* :v:

* clean up config

* typesを戻した

* Update packages/client/src/components/notification.vue

Co-authored-by: Acid Chicken (硫酸鶏) <root@acid-chicken.com>

* disconnect

* oops

* Failed to load the script unexpectedly回避
sw.jsとlib.tsを分離してみた

* truncate notification

* Update packages/client/src/ui/_common_/common.vue

Co-authored-by: syuilo <Syuilotan@yahoo.co.jp>

* clean up

* clean up

* キャッシュ対策

* Truncate push notification message

* クライアントがあったらストリームに接続しているということなので通知しない判定の位置を修正

* components/drive-file-thumbnail.vue

* components/drive-select-dialog.vue

* components/drive-window.vue

* merge

* fix

* Service Workerのビルドにesbuildを使うようにする

* return createEmptyNotification()

* fix

* i18n.ts

* update

* :v:

* remove ts-loader

* fix

* fix

* enhance: Service Workerを常に登録するように

* pollEnded

* URLをsw.jsに戻す

* clean up

* wip

* wip

* wip

* wip

* wip

* wip

* :v:

* use import

* fix

* install rollup

* use defineAsyncComponent.

* fix emojilist

* wip use defineAsyncComponent

* popup(import -> popup(defineAsyncComponent(() => import

* draggable?

* fix init import

* clean up

* fix router

* add comment

* :v:

* :v:

* :v:

* remove webpack

* update vite

* fix boot sequence

* Revert "fix boot sequence"

This reverts commit e893dbf37aed83bf9f12e427d98c78a7065b4a39.

* revert boot import

* never make two app div

* ;

* remove console.log

* change clientEntry sequence

* fix

* Revert "fix"

This reverts commit 12741b3d89950a31dbb1bb81477ddb27b0e9951a.

* fix

* add comment https://github.com/misskey-dev/misskey/pull/8575#issuecomment-1114239210

* add log

* add comment

Co-authored-by: Acid Chicken (硫酸鶏) <root@acid-chicken.com>
Co-authored-by: syuilo <Syuilotan@yahoo.co.jp>

dependabot[bot] committed:
chore(deps): bump postcss from 8.2.8 to 8.4.13 in /packages/client (#8588)

Bumps [postcss](https://github.com/postcss/postcss) from 8.2.8 to 8.4.13.
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/8.2.8...8.4.13)

---
updated-dependencies:
- dependency-name: postcss
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

pixeldesu committed:
fix(client): fix missing import of defineAsyncComponent in os.ts

tamaina committed:
Merge pull request #8593 from pixeldesu/fix/os-defineasync

Fix missing import of defineAsyncComponent in os.ts

pixeldesu committed:
refactor(client): refactor settings/accounts to use Composition API (#8604)

pixeldesu committed:
refactor(client): refactor settings/theme to use Composition API (#8595)

pixeldesu committed:
refactor(client): refactor settings/security to use Composition API (#8592)

pixeldesu committed:
refactor(client): refactor settings/plugin/install to use Composition API (#8591)

pixeldesu committed:
refactor(client): refactor settings/plugin to use Composition API (#8590)

pixeldesu committed:
Refactor settings/other to use Composition API (#8589)

* refactor(client): refactor settings/other to use Composition API

* fix(client): fix 'show featured notes' checkbox

pixeldesu committed:
Refactor settings/menu to use Composition API (#8586)

* refactor(client): refactor settings/menu to use Composition API

* Apply review suggestion from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

* Apply review suggestion from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

pixeldesu committed:
Refactor settings/general to use Composition API (#8578)

* refactor(client): refactor settings/general to use Composition API

* fix(client): turn watcher callback asynchronous

pixeldesu committed:
Refactor settings/email to use Composition API (#8576)

* refactor(client): refactor settings/email to use Composition API

* fix(client): switch to non-null assertion for settings values

pixeldesu committed:
refactor(client): refactor settings/drive to use Composition API (#8573)

pixeldesu committed:
Refactor delete-account to use Composition API (#8572)

* refactor(client): refactor delete-account to use Composition API

* Apply review suggestion from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

pixeldesu committed:
refactor(client): refactor settings/apps to use Composition API (#8570)

pixeldesu committed:
refactor(client): refactor settings/api to use Composition API (#8569)

pixeldesu committed:
Refactor account-info to use Composition API (#8568)

* refactor(client): refactor account-info  to use Composition API

* fix(client): use mounted hook for initial data

* fix(client): switch to non-null assertion for account check

pixeldesu committed:
Refactor settings/notifications to use Composition API (#8587)

* refactor(client): refactor settings/notifications to use Composition API

* fix(client): use async/await for API methods

pixeldesu committed:
fix(client): fix duplicate token request dialog in plugin install (#8612)

syuilo committed:
Update CONTRIBUTING.md

syuilo committed:
Update CONTRIBUTING.md

syuilo committed:
Update CONTRIBUTING.md

pixeldesu committed:
Fix lint issues in Drive components (#8613)

* fix(client): Fix lint issues in Drive components

* fix(client): only use !=/== for null comparisons

* Update drive.vue

Co-authored-by: syuilo <Syuilotan@yahoo.co.jp>

tamaina committed:
fix (client): fix mention icon height (#8615)

pixeldesu committed:
refactor(client): refactor 2FA settings to Composition API (#8599)

Johann150 committed:
chore: synchronize code and database schema (#8577)

* chore: remove default null

null is always the default value if a table column is nullable, and typeorm's
@Column only accepts strings for default.

* chore: synchronize code with database schema

* chore: sync generated migrations with code

pixeldesu committed:
Refactor settings/sounds to use Composition API (#8594)

* refactor(client): refactor settings/sounds to use Composition API

* Apply review suggestion from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

* chore(client): remove old sound reference

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

pixeldesu committed:
refactor(client): refactor settings/deck to use Composition API (#8598)

pixeldesu committed:
refactor(client): refactor settings/word-mute to use Composition API (#8597)

futchitwo committed:
Enhance(MFM): Allow speed changes in all animated MFMs (#8551)

* MFM: Allow speed changes in all animated MFMs

* Feature(MFM): Add speed property to cheat sheet

* Use template literal

Oops!

* Remove unnecessary template string

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

pixeldesu committed:
refactor(client): refactor settings/theme/manage to use Composition API (#8596)

pixeldesu committed:
fix(client): fix lint issues in scripts (#8621)

pixeldesu committed:
Fix lint issues in emoji picker components (#8620)

* fix(client): fix lint issues in emoji picker components

* fix(client): switch argument naming for emoji picker section event

pixeldesu committed:
Fix lint issues in post form component (#8619)

* fix(client): fix lint issues in post form

* Apply review suggestions from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

pixeldesu committed:
fix(client): add setup attribute to notification page (#8648)

syuilo committed:
New Crowdin updates (#8488)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Chinese Simplified)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Romanian)

* New translations ja-JP.yml (Romanian)

* New translations ja-JP.yml (Romanian)

* New translations ja-JP.yml (Romanian)

* New translations ja-JP.yml (Romanian)

* New translations ja-JP.yml (Romanian)

* New translations ja-JP.yml (Ukrainian)

* New translations ja-JP.yml (Ukrainian)

* New translations ja-JP.yml (Arabic)

* New translations ja-JP.yml (Catalan)

* New translations ja-JP.yml (Catalan)

* New translations ja-JP.yml (Catalan)

* New translations ja-JP.yml (Catalan)

* New translations ja-JP.yml (Catalan)

* New translations ja-JP.yml (Catalan)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Arabic)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Czech)

* New translations ja-JP.yml (Czech)

* New translations ja-JP.yml (Czech)

* New translations ja-JP.yml (Spanish)

* New translations ja-JP.yml (Spanish)

* New translations ja-JP.yml (Spanish)

* New translations ja-JP.yml (Spanish)

* New translations ja-JP.yml (Spanish)

* New translations ja-JP.yml (Spanish)

* New translations ja-JP.yml (Spanish)

* New translations ja-JP.yml (Arabic)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Portuguese)

* New translations ja-JP.yml (Portuguese)

* New translations ja-JP.yml (Arabic)

* New translations ja-JP.yml (Russian)

* New translations ja-JP.yml (Kabyle)

* New translations ja-JP.yml (Kannada)

* New translations ja-JP.yml (Bengali)

* New translations ja-JP.yml (Indonesian)

* New translations ja-JP.yml (English)

* New translations ja-JP.yml (Chinese Simplified)

* New translations ja-JP.yml (Slovak)

* New translations ja-JP.yml (Polish)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Dutch)

* New translations ja-JP.yml (Korean)

* New translations ja-JP.yml (Italian)

* New translations ja-JP.yml (German)

* New translations ja-JP.yml (French)

* New translations ja-JP.yml (Romanian)

* New translations ja-JP.yml (Portuguese)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Spanish)

* New translations ja-JP.yml (Czech)

* New translations ja-JP.yml (Catalan)

* New translations ja-JP.yml (Ukrainian)

* New translations ja-JP.yml (Japanese, Kansai)

* New translations ja-JP.yml (German)

* New translations ja-JP.yml (English)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Arabic)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Slovak)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Korean)

* New translations ja-JP.yml (Korean)

* New translations ja-JP.yml (Chinese Simplified)

* New translations ja-JP.yml (Chinese Simplified)

* New translations ja-JP.yml (Chinese Simplified)

* New translations ja-JP.yml (Chinese Simplified)

* New translations ja-JP.yml (Chinese Simplified)

* New translations ja-JP.yml (Chinese Simplified)

* New translations ja-JP.yml (Chinese Simplified)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Indonesian)

* New translations ja-JP.yml (German)

* New translations ja-JP.yml (Korean)

* New translations ja-JP.yml (Indonesian)

* New translations ja-JP.yml (Arabic)

Johann150 committed:
perf: fix caching (#8660)

The cache implementation did previously not store the results of the
computation and was thus not a cache at all. This can cause a significant
number of database queries each time someone with a large number of
followers does something that causes an activity to be federated.

syuilo committed:
update deps

syuilo committed:
enhance: Display TOTP Register URL

Close #7261

Co-Authored-By: tamaina <tamaina@hotmail.co.jp>

Johann150 committed:
fix: keep file order (#8659)

ishowta committed:
test: e2eテストがCIで失敗していた問題をいくつか修正 (#8642)

* test: indexeddbをテスト毎に初期化するように

* fix: metaが無いときにfetch-metaを同時に呼ぶと死ぬことがある問題を修正

* test: ログイン後のクライアント側処理を待たずにリロードされてログイン出来ないことがあったのを修正

tamaina committed:
fix: ユーザー検索で、クエリがusernameの条件を満たす場合はusernameもLIKE検索するように (#8644)

* Fix #8643

* 部分一致にする

ishowta committed:
test: `__dirname`はESModuleでは使えないので置き換えた (#8626)

ishowta committed:
test: Nodeのカスタムローダーを直してテストが動くように (#8625)

* test: Nodeのカスタムローダーを直してテストが動くように

* dev: mochaを呼ぶコマンドにNODE_ENV=testを追加

* Update packages/backend/test/loader.js

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

* chore: change export style in loader.js

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

syuilo committed:
Update extensions.json

pixeldesu committed:
Refactor Chart component (#8622)

* refactor(client): refactor Chart component

* Apply review suggestions from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

* fix(client): don't expose values from Chart

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

pixeldesu committed:
refactor(client): refactor admin/ads to use Composition API (#8649)

pixeldesu committed:
refactor(client): refactor admin/announcements to use Composition API (#8650)

pixeldesu committed:
Refactor admin/security to use Composition API (#8652)

* refactor(client): refactor admin/security to use Composition API

* Apply review suggestions from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

pixeldesu committed:
Refactor admin/database to use Composition API (#8654)

* refactor(client): refactor admin/database to use Composition API

* Apply review suggestion from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

pixeldesu committed:
Refactor admin/email-settings to use Composition API (#8656)

* refactor(client): refactor admin/email-settings to use Composition API

* Update packages/client/src/pages/admin/email-settings.vue

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Co-authored-by: syuilo <Syuilotan@yahoo.co.jp>
Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

pixeldesu committed:
fix(client): remove unexpected token (#8672)

syuilo committed:
fix(server): prevent crash when processing certain PNGs

Fix #8605

syuilo committed:
feat: make captcha required when signin to improve security

syuilo committed:
feat(dev): okteto integration

syuilo committed:
CAPTCHA求めるのは2fa認証が無効になっているときだけにした

2faのトークンは期限付きだから、CAPTCHA解いてる間に期限切れになる

syuilo committed:
ビルド関係の依存関係をdevDependenciesに入れるのをやめた

本番環境でビルドできないため

Johann150 committed:
fix: postgres type error

Fix a bug introduced in #8659. Solution was already tested there.

Johann150 committed:
chore: update changelog

pixeldesu committed:
Refactor emoji-edit-dialog to use Composition API (#8657)

* refactor(client): refactor emoji-edit-dialog to use Composition API

* fix(client): fix editing emoji not updating emoji list

* Apply review suggestions from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

* fix(client): use cached category info instead of making a request

* fix(client): use updateItem in emoji pagination when editing

* fix(client): reimplement removeItem in MkPagination

* Apply review suggestion from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

ikuradon committed:
Power up (#8684)

syuilo committed:
chore(dev): use .yaml for prevent okteto error

ThatOneCalculator committed:
Improve README (#8645)

* Add files via upload

* Update title.svg

* Floating title!

* Update README.md

* Update title_float.svg

* Consolidate taglines

* Update README.md

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

* Update README.md

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

* Update README.md

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

* Update README.md

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

* Update README.md

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

* Update README.md

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

* Update README.md

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

* Update README.md

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

* Update README.md

* Drive

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

pixeldesu committed:
refactor(client): refactor my-antennas/index to use Composition API (#8679)

pixeldesu committed:
Refactor admin/relays to use Composition API (#8677)

* refactor(client): refactor admin/relays to use Composition API

* Apply review suggestion from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

pixeldesu committed:
Refactor admin/queue to use Composition API (#8676)

* refactor(client): refactor admin/queue to use Composition API

* Apply review suggestion from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

pixeldesu committed:
refactor(client): refactor admin/proxy-account to use Composition API (#8675)

pixeldesu committed:
refactor(client): refactor admin/object-storage to use Composition API (#8666)

pixeldesu committed:
Refactor admin/integrations to use Composition API (#8664)

* refactor(client): refactor admin/integrations to use Composition API

* Apply review suggestions from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

pixeldesu committed:
refactor(client): refactor admin/instance-block to use Composition API (#8663)

pixeldesu committed:
Refactor admin/index to use Composition API (#8662)

* refactor(client): refactor admin/index to use Composition API

* fix(client): fix navigation to initial admin pages

* Apply review suggestions from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

* fix(client): re-add abuses page to admin/index

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

pixeldesu committed:
Refactor file-dialog to use Composition API (#8661)

* refactor(client): refactor file-dialog to use Composition API

* Apply review suggestion from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

adzukyat committed:
remove unneeded attrs (#8673)

mei23 committed:
enhance: Perform port diagnosis at startup only when Listen fails (#8698)

* Change port check

* Comment: disableClustering

* CHANGELOG

* Smart message

dependabot[bot] committed:
chore(deps): bump path-parse from 1.0.6 to 1.0.7 (#8705)

Bumps [path-parse](https://github.com/jbgutierrez/path-parse) from 1.0.6 to 1.0.7.
- [Release notes](https://github.com/jbgutierrez/path-parse/releases)
- [Commits](https://github.com/jbgutierrez/path-parse/commits/v1.0.7)

---
updated-dependencies:
- dependency-name: path-parse
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump async from 3.2.0 to 3.2.3 in /packages/backend (#8706)

Bumps [async](https://github.com/caolan/async) from 3.2.0 to 3.2.3.
- [Release notes](https://github.com/caolan/async/releases)
- [Changelog](https://github.com/caolan/async/blob/master/CHANGELOG.md)
- [Commits](https://github.com/caolan/async/compare/v3.2.0...v3.2.3)

---
updated-dependencies:
- dependency-name: async
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

syuilo committed:
Update CONTRIBUTING.md

syuilo committed:
chore(client): tweak loading spinner design

xianonn committed:
fix: ノートのインスタンス情報の文字に縁を付けて見やすくする (#8697)

* ノートのインスタンス情報の背景色が反映されないことがあるのを修正する

* ノートのインスタンス情報の文字に縁を付けて見やすくする

* Revert "ノートのインスタンス情報の背景色が反映されないことがあるのを修正する"

This reverts commit de920dfc537d1f2c68804d0d6930520f2b3cbce7.

* ノートのインスタンス情報の文字の影の数を増やしてさらに見やすくする

mei23 committed:
fix: Unable to generate video thumbnails (#8696)

* fix: Unable to generate video thumbnails

* CHANGELOG

Johann150 committed:
enhance: uniform theme color (#8702)

* enhance: make theme color format uniform

All newly fetched instance theme colors will be uniformely formatted
as hashtag followed by 6 hexadecimal digits.

Colors are checked for validity and invalid colors are not handled.

* better input validation for own theme color

* migration to unify theme color formats

Fixes theme colors of other instances as well as the local instance.

* add changelog entry

Co-authored-by: syuilo <Syuilotan@yahoo.co.jp>

syuilo committed:
chore(client): tweak loading spinner design

syuilo committed:
Merge branch 'develop' of https://github.com/misskey-dev/misskey into develop

dependabot[bot] committed:
chore(deps): bump async from 3.2.1 to 3.2.3 in /packages/client (#8707)

Bumps [async](https://github.com/caolan/async) from 3.2.1 to 3.2.3.
- [Release notes](https://github.com/caolan/async/releases)
- [Changelog](https://github.com/caolan/async/blob/master/CHANGELOG.md)
- [Commits](https://github.com/caolan/async/compare/v3.2.1...v3.2.3)

---
updated-dependencies:
- dependency-name: async
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

pixeldesu committed:
fix(client): fix lint issues in Deck UI components (#8681)

dependabot[bot] committed:
chore(deps): bump async from 3.2.1 to 3.2.3 (#8501)

Bumps [async](https://github.com/caolan/async) from 3.2.1 to 3.2.3.
- [Release notes](https://github.com/caolan/async/releases)
- [Changelog](https://github.com/caolan/async/blob/master/CHANGELOG.md)
- [Commits](https://github.com/caolan/async/compare/v3.2.1...v3.2.3)

---
updated-dependencies:
- dependency-name: async
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump copy-props from 2.0.4 to 2.0.5 (#8709)

Bumps [copy-props](https://github.com/gulpjs/copy-props) from 2.0.4 to 2.0.5.
- [Release notes](https://github.com/gulpjs/copy-props/releases)
- [Changelog](https://github.com/gulpjs/copy-props/blob/master/CHANGELOG.md)
- [Commits](https://github.com/gulpjs/copy-props/compare/2.0.4...2.0.5)

---
updated-dependencies:
- dependency-name: copy-props
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

pixeldesu committed:
Refactor pleaseLogin to show a sign-in dialog (#8630)

* refactor(client): refactor pleaseLogin to show a sign-in dialog

* Apply review suggestions from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>
Co-authored-by: syuilo <Syuilotan@yahoo.co.jp>

pixeldesu committed:
refactor(client): refactor admin/settings to use Composition API (#8678)

xianonn committed:
fix: ノート詳細ページの新しいノートを表示する機能の動作が正しくなるように修正する (#8607)

* ノート詳細で新しいノートの表示が正しくないのを修正する

* ノート詳細から別のノート詳細を表示した時に前後の表示をリセットする

Johann150 committed:
enhance(MFM): limit large MFM (#8540)

* add CSS classes for zoom MFM

* limit nesting of x2, x3, x4 MFM

* simplify CSS calculation

Co-authored-by: Acid Chicken (硫酸鶏) <root@acid-chicken.com>

Co-authored-by: Acid Chicken (硫酸鶏) <root@acid-chicken.com>

Johann150 committed:
fix(activitypub): add authorization checks (#8534)

* fix spelling

* fix(activitypub): add authorization checks

Johann150 committed:
enhance: page image component with alt text (#8634)

* refactor to composition API

* use existing image component

This improves user experience because alt text is displayed correctly.

* fix: correct image src

* fix: defineProps

* fix

Johann150 committed:
hotfix: uniform color migration fix

syuilo committed:
fix(client): make emoji stand out more on reaction button

Fix #8520
Close #8521

Co-Authored-By: Johann150 <20990607+Johann150@users.noreply.github.com>

Johann150 committed:
chore: remove unused imports

pixeldesu committed:
chore(meta): add pixeldesu to patron list (#8714)

syuilo committed:
feat(dev): introduce Pull Request Labeler

syuilo committed:
Merge branch 'develop' of https://github.com/misskey-dev/misskey into develop

dependabot[bot] committed:
chore(deps): bump hosted-git-info from 2.8.8 to 2.8.9 (#8708)

Bumps [hosted-git-info](https://github.com/npm/hosted-git-info) from 2.8.8 to 2.8.9.
- [Release notes](https://github.com/npm/hosted-git-info/releases)
- [Changelog](https://github.com/npm/hosted-git-info/blob/v2.8.9/CHANGELOG.md)
- [Commits](https://github.com/npm/hosted-git-info/compare/v2.8.8...v2.8.9)

---
updated-dependencies:
- dependency-name: hosted-git-info
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

syuilo committed:
Update utils.ts

syuilo committed:
Update utils.ts

syuilo committed:
refactor

syuilo committed:
lint

syuilo committed:
Update .mocharc.json

syuilo committed:
chore(dev): tweak text

pixeldesu committed:
fix(client): fix undefined data value on 2FA settings (#8725)

Johann150 committed:
fix(client): wrong scoping breaks 2FA

ThatOneCalculator committed:
Update README.md (#8729)

Johann150 committed:
fix: wrong type for isVisibleForMe

Johann150 committed:
feat(dev): highlight editing of wrong locales

Highlight PRs that edit locales other than the ja-JP one so the author may see and fix it themselves.

mei23 committed:
Supports Unicode Emoji 14.0 (#8699)

* Unicode 14.0 Emoji

* mfm-js@0.22.0

* CHANGELOG

Co-authored-by: syuilo <Syuilotan@yahoo.co.jp>

pixeldesu committed:
Refactor my-antennas/edit to use Composition API (#8680)

* refactor(client): refactor my-antennas/edit to use Composition API

* fix(client): apply review suggestions

pixeldesu committed:
Refactor admin/overview to use Composition API (#8674)

* refactor(client): refactor admin/overview to use Composition API

* fix(client): apply review suggestions

pixeldesu committed:
refactor(client): refactor admin/other-settings to use Composition API (#8667)

pixeldesu committed:
Refactor widgets and fix lint issues (#8719)

* fix(client): refactor widgets and fix lint issues

* Apply review suggestions from @Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Johann150 committed:
refactor: temporary files (#8713)

* simplify temporary files for thumbnails

Because only a single file will be written to the directory, creating a
separate directory seems unnecessary. If only a temporary file is created,
the code from `createTemp` can be reused here as well.

* refactor: deduplicate code for temporary files/directories

To follow the DRY principle, the same code should not be duplicated
across different files. Instead an already existing function is used.

Because temporary directories are also create in multiple locations,
a function for this is also newly added to reduce duplication.

* fix: clean up identicon temp files

The temporary files for identicons are not reused and can be deleted
after they are fully read. This condition is met when the stream is closed
and so the file can be cleaned up using the events API of the stream.

* fix: ensure cleanup is called when download fails

* fix: ensure cleanup is called in error conditions

This covers import/export queue jobs and is mostly just wrapping all
code in a try...finally statement where the finally runs the cleanup.

* fix: use correct type instead of `any`

Johann150 committed:
fix: server metrics widget

Johann150 committed:
fix: activity widget used wrong variable name

Johann150 committed:
fix: assume remote users are following each other (#8734)

Misskey does not know if two remote users are following each other.
Because ActivityPub actions would otherwise fail on followers only
notes, we have to assume that two remote users are following each other
when an interaction about a remote note occurs.

syuilo committed:
refactor: use ===

syuilo committed:
update deps

Johann150 committed:
use http-signature module that supports hs2019 (#8635)

Johann150 committed:
fix lints (#8737)

* fix: emits use ev instead of e

* fix: errors use err instead of e

* fix: replace use of data where possible

* fix: events use evt instead of e

* fix: use strict equals

* fix: use emoji instead of e

* fix: vue lints

Johann150 committed:
fix(docs): correct information for drive upload (#8736)

Johann150 committed:
enhance: clearly link documentation

fix #8744

Johann150 committed:
enhance: replace signin CAPTCHA with rate limit (#8740)

* enhance: rate limit works without signed in user

* fix: make limit key required for limiter

As before the fallback limiter key will be set from the endpoint name.

* enhance: use limiter for signin

* Revert "CAPTCHA求めるのは2fa認証が無効になっているときだけにした"

This reverts commit 02a43a310f6ad0cc9e9beccc26e51ab5b339e15f.

* Revert "feat: make captcha required when signin to improve security"

This reverts commit b21b0580058c14532ff3f4033e2a9147643bfca6.

* fix undefined reference

* fix: better error message

* enhance: only handle prefix of IPv6

ThatOneCalculator committed:
Remove require captcha from signin from CHANGELOG (#8748)

pixeldesu committed:
feat(tests): add e2e tests for widgets (#8735)

* test(e2e): add baseline for widget tests

* chore(repo): enable test running in branch

* fix(e2e): set viewport for widget tests

* fix(client): add widget identifier classes to widgets

* test(e2e): add memo widget test

* fix(tests): force select value

* fix(tests): force button press for widget addition

* fix(tests): invoke select value differently

* fix(tests): adjust widget submit

* fix(tests): don't explicitly navigate for widget test

* fix(tests): click label to hide select popup

* fix(tests): just click modal background

* fix(tests): adjust modal background selector

* fix(tests): click all modal backgrounds

* feat(e2e): add test for adding timeline widget

* fix(client): add more widget identifier classes

* feat(tests): add method abstraction for test cases

* fix(tests): force-click overlays

* fix(tests): force widget button press

* fix(tests): remove timeout from final widget check

* feat(tests): add widget removal test case

* fix(client): use mk instead of msky as class prefix

* fix(tests): check widgets for existence rather than visibility

* chore(meta): don't run tests for specific feature branch

tamaina committed:
Fix theme import (#8749)

tamaina committed:
refactor: use css module at components/global/loading.vue (#8750)

* refactor: use css module at components/global/loading.vue

* rename class name to "root"

Johann150 committed:
fix: validate text is not empty

fix #8747

tamaina committed:
preload app css (#8752)

tamaina committed:
fix(client): Vite related boot mechanism revision (#8753)

* preload app css

* remove salt

* APP_FETCH_FAILED error

* set max-age to 15s

Johann150 committed:
refactor: improve code quality (#8751)

* remove unnecessary if

`Array.prototype.some` already returns a boolean so an if to return
true or false is completely unnecessary in this case.

* perf: use count instead of find

When using `count` instead of `findOneBy`, the data is not
unnecessarily loaded.

* remove duplicate null check

The variable is checked for null in the lines above and the function
returns if so. Therefore, it can not be null at this point.

* simplify `getJsonSchema`

Because the assigned value is `null` and the used keys are only
shallow, use of `nestedProperty.set` seems inappropriate. Because the
value is not read, the initial for loop can be replaced by a `for..in`
loop.

Since all keys will be assigned `null`, the condition of the ternary
expression in the nested function will always be true. Therefore the
recursion case will never happen. With this the nested function can be
eliminated.

* remove duplicate condition

The code above already checks `dragging` and returns if it is truthy.
Checking it again later is therefore unnecessary.

To make this more obvious the `return` is removed in favour of using
an if...else construct.

* remove impossible "unknown" time

The `ago` variable will always be a number and all non-negative numbers
are already covered by other cases, the negative case is handled with
`future` so there is no case when `unkown` could be achieved.

futchitwo committed:
fix(client): fix popout url (#8494)

Johann150 committed:
enhance(dev): ask for log snippets

Johann150 committed:
fix: add missing import

fix #8756

pixeldesu committed:
fix(client): import shared ESLint config in client package (#8761)

tamaina committed:
chore: remove packages/sw/webpack.config.js

tamaina committed:
Merge branch 'develop' of https://github.com/misskey-dev/misskey into develop

tamaina committed:
chore: fix import tinycolor

pixeldesu committed:
chore(meta): label Pull Requests containing tests (#8768)

mei23 committed:
Fix IP address rate limit (#8758)

* Fix IP address rate limit

* CHANGELOG

* Tune getIpHash

Johann150 committed:
fix: always remove completed tasks (#8771)

mei23 committed:
Fix `Cannot find module` issue (#8770)

* Add --force to yarn in the installation script

* CHAGELOG

pixeldesu committed:
Extract commonly used test logic to commands (#8767)

* meta(tests): enable workflows to run in branch

* feat(tests): move commonly used logic to Cypress commands

* chore(tests): replace more code with commands

* meta(tests): disable workflows to run in branch

Johann150 committed:
fix(mfm): remove duplicate br tag/newline (#8616)

Johann150 committed:
fix(lint): indentation

Johann150 committed:
fix: server metrics widget

Johann150 committed:
fix(dev): no labels for l10n_develop

Commit:
fix(client): correctly handle MiAuth URLs with query string (#8772)

syuilo committed:
fix(test): reset redis in e2e test

#7986

syuilo committed:
Merge branch 'develop' of https://github.com/misskey-dev/misskey into develop

PikaDude committed:
User moderation details (#8762)

* add more user details for admins to see

* fix some issues

* small style fix

as suggested by Johann150

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

* fix

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Co-authored-by: Johann150 <johann@qwertqwefsday.eu>

Johann150 committed:
fix: correctly render empty note text (#8746)

Ensure that the _misskey_content attribute will always exist. Because
the API endpoint does not require the existence of the `text` field,
that field may be `undefined`. By using `?? null` it can be ensured
that the value is at least `null`.

Furthermore, the rendered HTML of a note with empty text will also be
the empty string. From git blame it seems that this behaviour was added
because of a Mastodon bug that might have previously existed. Hoever,
this seems to be no longer the case as I can find mastodon posts that
have empty content.

The code could be made a bit more succinct by using the null coercion
operator.

Johann150 committed:
fix: ensure resolver does not fetch local resources via HTTP(S) (#8733)

* refactor: parseUri types and checks

The type has been refined to better represent what it actually is. Uses of
parseUri are now also checking the parsed object type before resolving.

* cannot resolve URLs with fragments

* also take remaining part of URL into account

Needed for parsing the follows URIs.

* Resolver uses DbResolver for local

* remove unnecessary use of DbResolver

Using DbResolver would mean that the URL is parsed and handled again.
This duplicated processing can be avoided by querying the database directly.

* fix missing property name

Johann150 committed:
fix: add id for activitypub follows (#8689)

* add id for activitypub follows

* fix lint

* fix: follower must be local, followee must be remote

Misskey will only use ActivityPub follow requests for users that are local
and are requesting to follow a remote user. This check is to ensure that
this endpoint can not be used by other services or instances.

* fix: missing import

* render block with id

* fix comment

yitsushi committed:
feat: option to collapse long notes (#8561)

* feat: option to collapse long notes

Closes #8559

* do not collapse if cw exists

* use '閉じる' to close / show less.

* make it sticky

* Change style of the Show less button

syuilo committed:
fix test

syuilo committed:
fix bug

syuilo committed:
Revert "feat: option to collapse long notes (#8561)"

This reverts commit e675ffcf38b07f5c70d00b49c171c7ab3460e810.

syuilo committed:
Update CHANGELOG.md

syuilo committed:
Update CONTRIBUTING.md

syuilo committed:
Update CONTRIBUTING.md

syuilo committed:
Update CHANGELOG.md

syuilo committed:
update summaly

syuilo committed:
use node 16

syuilo committed:
chore(client): fix menu item style

syuilo committed:
chore: lint fixes

syuilo committed:
Update .mocharc.json

syuilo committed:
fix(test): make chart tests working

syuilo committed:
Update .node-version

Johann150 committed:
refactor: use awaitAll to reduce duplication (#8791)

* refactor: use awaitAll to reduce duplication

* fix lint

* fix typo

Johann150 committed:
fix: try to prevent autocomplete for emoji search (#8798)

syuilo committed:
Update CONTRIBUTING.md

ikuradon committed:
ok-to-test with okteto (#8799)

syuilo committed:
chore(dev): update okteto workflow

syuilo committed:
Update pr-preview-deploy.yml

syuilo committed:
chore(dev): update okteto workflow

tamaina committed:
Display the deploy status on checks (#8803)

* Display deploy status on check suite

* Display deploy status on check suite

* fix

* fix

syuilo committed:
Update CONTRIBUTING.md

Johann150 committed:
properly display alt text for videos (#8802)

The alt text is already properly federated and given by the API,
it is just not displayed properly by the client.

Johann150 committed:
fix: use autocomplete=new-password (#8797)

pixeldesu committed:
fix(client): render quote renote CWs as MFM (#8792)

Co-authored-by: syuilo <Syuilotan@yahoo.co.jp>

syuilo committed:
Update CHANGELOG.md

Johann150 committed:
enhance: improve documentation for `/users/` endpoints (#8790)

* docs: category & description for reset password

* docs: category & description for testing

* docs: descriptions for groups endpoints

* docs: descriptions for drive file endpoints

* docs: descriptions for sw endpoints

* docs: descriptions for user list endpoints

* docs: descriptions & result type for gallery posts

* docs: descriptions & result type for user endpoints

* docs: add return type for stats

Johann150 committed:
refactor: follow button (#8789)

* fix: display cancelling follow request

* remove unnecessary branch

The executed code is the same as in the else branch so this special
condition is unnecessary.

* remove code duplication

Use the same callback as later for updating these variables.

* use $ref sugar

* remove unused import

Co-authored-by: blackskye-sx <saul.newman@gmail.com>

Johann150 committed:
enhance(federation): use ActivityPub defined property in favour of proprietary property. (#8787)

* add activitypub `source` property

* parse MFM from new `source` attribute

Johann150 committed:
chore: fix some lints automatically (#8788)

* chore: fix some lints automatically

Fixed lints that were automatically fixable with `eslint --fix`.

* fix type

* workaround for empty interface lint

syuilo committed:
chore: tweak logo

syuilo committed:
Merge branch 'develop' of https://github.com/misskey-dev/misskey into develop

syuilo committed:
update deps

syuilo committed:
Merge branch 'master' into develop

syuilo committed:
update cypress

syuilo committed:
update cypress

syuilo committed:
update cypress

syuilo committed:
feat: image cropping (#8808)

* wip

* wip

* wip

syuilo committed:
update cypress

syuilo committed:
fix(test): use cypress-io/github-action@v4

syuilo committed:
New Crowdin updates (#8669)

* New translations ja-JP.yml (Chinese Simplified)

* New translations ja-JP.yml (German)

* New translations ja-JP.yml (English)

* New translations ja-JP.yml (German)

* New translations ja-JP.yml (English)

* New translations ja-JP.yml (Catalan)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Indonesian)

* New translations ja-JP.yml (Slovak)

* New translations ja-JP.yml (French)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Russian)

* New translations ja-JP.yml (Russian)

* New translations ja-JP.yml (Korean)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (English)

* New translations ja-JP.yml (German)

* New translations ja-JP.yml (Indonesian)

* New translations ja-JP.yml (Italian)

* New translations ja-JP.yml (Italian)

* New translations ja-JP.yml (Arabic)

* New translations ja-JP.yml (Russian)

* New translations ja-JP.yml (Bengali)

* New translations ja-JP.yml (Indonesian)

* New translations ja-JP.yml (English)

* New translations ja-JP.yml (Chinese Simplified)

* New translations ja-JP.yml (Slovak)

* New translations ja-JP.yml (Polish)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Korean)

* New translations ja-JP.yml (Italian)

* New translations ja-JP.yml (German)

* New translations ja-JP.yml (French)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Spanish)

* New translations ja-JP.yml (Ukrainian)

* New translations ja-JP.yml (Japanese, Kansai)

* New translations ja-JP.yml (Slovak)

* New translations ja-JP.yml (Bengali)

* New translations ja-JP.yml (Bengali)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Swedish)

* New translations ja-JP.yml (Arabic)

* New translations ja-JP.yml (Slovak)

* New translations ja-JP.yml (Japanese, Kansai)

* New translations ja-JP.yml (Bengali)

* New translations ja-JP.yml (Indonesian)

* New translations ja-JP.yml (English)

* New translations ja-JP.yml (Chinese Simplified)

* New translations ja-JP.yml (Russian)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Romanian)

* New translations ja-JP.yml (Ukrainian)

* New translations ja-JP.yml (Czech)

* New translations ja-JP.yml (Spanish)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (French)

* New translations ja-JP.yml (Polish)

* New translations ja-JP.yml (German)

* New translations ja-JP.yml (Italian)

* New translations ja-JP.yml (Korean)

* New translations ja-JP.yml (Arabic)

* New translations ja-JP.yml (Vietnamese)

* New translations ja-JP.yml (Bengali)

* New translations ja-JP.yml (Indonesian)

* New translations ja-JP.yml (English)

* New translations ja-JP.yml (Chinese Simplified)

* New translations ja-JP.yml (Slovak)

* New translations ja-JP.yml (Russian)

* New translations ja-JP.yml (Polish)

* New translations ja-JP.yml (Ukrainian)

* New translations ja-JP.yml (Czech)

* New translations ja-JP.yml (Spanish)

* New translations ja-JP.yml (Chinese Traditional)

* New translations ja-JP.yml (Romanian)

* New translations ja-JP.yml (French)

* New translations ja-JP.yml (German)

* New translations ja-JP.yml (Italian)

* New translations ja-JP.yml (Korean)

* New translations ja-JP.yml (Japanese, Kansai)

* New translations ja-JP.yml (Portuguese)

* New translations ja-JP.yml (Portuguese)

Johann150 committed:
chore: synchronize visibility checks (#8687)

* reuse single meId parameter

* unify code style

Use template string to avoid having to use escaped quote marks.

* fix: follower only notes are visible to mentioned users

This synchronizes the visibility rules with the Notes.isVisibleForMe
method from packages/backend/src/models/repositories/note.ts

* add comment

syuilo committed:
fix(client): オブジェクトストレージを使用していると画像のクロップができない

syuilo committed:
Merge branch 'develop' of https://github.com/misskey-dev/misskey into develop

syuilo committed:
Update cropper-dialog.vue

syuilo committed:
Update cropper-dialog.vue

syuilo committed:
New Crowdin updates (#8809)

* New translations ja-JP.yml (Dutch)

* New translations ja-JP.yml (German)

* New translations ja-JP.yml (English)

* New translations ja-JP.yml (Indonesian)

syuilo committed:
Update CHANGELOG.md

syuilo committed:
12.111.0

syuilo committed:
Merge pull request #8783 from misskey-dev/develop

Release: 12.111.0

futchitwo committed:
Improve: unclip

futchitwo committed:
Fix unclip

futchitwo committed:
Fix unclip

futchitwo committed:
chore: better type and name

futchitwo committed:
Refactor clip page to use Composition API

futchitwo committed:
Refactor clip page

futchitwo committed:
Refactor clip page

futchitwo committed:
Refactor clip page

futchitwo committed:
Fix

tamaina committed:
fix: some fixes of multiple notification read (#8819)

* fix: limit multiple notification read

* fix

* fix

syuilo committed:
fix(client): デッキでウィジェットの情報が保存されない問題を修正

Fix #8818

syuilo committed:
lint

futchitwo committed:
Fix(client): fix wrong gallery page component (#8815)

syuilo committed:
New translations ja-JP.yml (Korean) (#8812)

syuilo committed:
Update CHANGELOG.md

syuilo committed:
Merge branch 'develop' of https://github.com/misskey-dev/misskey into develop

futchitwo committed:
Fix clipPage vue provider

Johann150 committed:
fix: missing file name parameter (#8820)

syuilo committed:
12.111.1

syuilo committed:
Merge branch 'master' into develop

ghost commented:
Your preview environment [futchitwo-misskey-pr-151](https://cloud.okteto.com/#/previews/futchitwo-misskey-pr-151) has been deployed.

Preview environment endpoints are available at:

- [https://misskey-futchitwo-misskey-pr-151.cloud.okteto.net](https://misskey-futchitwo-misskey-pr-151.cloud.okteto.net)

syuilo committed:
lint fixes

Johann150 committed:
fix: remove unused parameter

Johann150 committed:
fix(docs): use correct description property

tamaina committed:
fix: add limit to i/notifications (#8836)

* fix: add limit to i/notifications

* ms

* remove ms

syuilo committed:
enhance(server): モデレーターであってもレートリミットを有効に

Johann150 committed:
fix: tmpdir cleanup removes contained files (#8826)

mei23 committed:
fix: GenerateVideoThumbnail (#8825)

* fix: GenerateVideoThumbnail

* CHANGELOG

* fix cleanup

* Revert "fix cleanup"

This reverts commit d54cf8262ac01a3deb6b8dd7689ec144d4d09ea8.

syuilo committed:
Update CHANGELOG.md

syuilo committed:
enhance(client): improve files page of control panel

Johann150 committed:
fix: correctly render note text

Fix a regression from #8787 that was previously fixed in #8440.

Johann150 committed:
fix: render empty note content correctly

Instead of coercing to `null`, coercing to an empty string should simplify handling.

syuilo committed:
Update CONTRIBUTING.md

syuilo committed:
chore(client): tweak range control design

syuilo committed:
Update ROADMAP.md

syuilo committed:
Merge branch 'develop' of https://github.com/misskey-dev/misskey into develop

syuilo committed:
enhance(client): improve file moderation ui

mei23 committed:
enhance: Improve player detection in URL preview (#8849)

* enhance: Improve player detection in URL preview

* CHANGELOG

futchitwo committed:
Refactor clip page to Composition API (#8822)

* Refactor clip page to use Composition API

* Refactor clip page

* Refactor clip page

* Refactor clip page

syuilo committed:
Merge branch 'develop' into clip-remove-rebase
